File: src/main/java/dev/naman/lldassignments/designpatterns/decorator/texteditor/solution/SimpleTextEditor.java
package dev.naman.lldassignments.designpatterns.decorator.texteditor.solution;

class SimpleTextEditor extends TextEditor {
    private final String text;

    public SimpleTextEditor(String text) {
        this.text = text;
    }

    @Override
    public String getText() {
        return text;
    }
}

File: src/main/java/dev/naman/lldassignments/designpatterns/decorator/texteditor_csharp/SimpletextEditor.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TextEditor
    {
        public class SimpletextEditor:textEditor
        {
            private string text;
            public SimpletextEditor(string t)
            {
                text = t;
            }


            public override string getText()
                {
                    return this.text;
                }

            public void setText(string text)
                {
                    this.text = text;
                }
        }
    }
